src/ChatFeed/MessageForm/Attachment/styles.tsx

```diff
@@ -0,0 +1,47 @@
+import { Properties } from 'csstype';
+
+export interface AttachmentStyles {
+  attachmentWrapper?: Properties;
+  fileAttachmentPreview?: Properties;
+  imageAttachmentPreview?: Properties;
+  removeAttachmentIcon?: Properties;
+}
+
+export const styles = {
```

Comment:
add the type declaration

`export const styles: AttachmentStyles = {`

Reply:
Ahhhhh i knew i was doing something off - thanks java lord 😉